c7n/filters/__init__.py

```diff
@@ -21,5 +21,5 @@
 from .iamaccess import CrossAccountAccessFilter, PolicyChecker
 from .iamanalyzer import AccessAnalyzer
 from .metrics import MetricsFilter, ShieldMetrics
-from .vpc import DefaultVpcBase
+# from .vpc import DefaultVpcBase
```

Comment:
nit: Commented just for testing? Guessing we can nuke this line?

Reply:
nuke is good, was commented to find all the spots that failed, but those are covered at this point.